### PR TITLE
Add Kibana dashboards for system module

### PR DESCRIPTION
**Pull request by monicasarbu**
This PR adds multiple Kibana dashboards that display data collected by the system module of Metricbeat. The dashboards are connected together using the navigation bar.

### CODE REVIEW

**Comment by monicasarbu** [comment]
metricbeat memory in kibana

File: metricbeat/etc/kibana/visualization/Avg.-Total-Memory.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "visState": "{\"title\":\"Avg. Total Memory\",\"type\":\"metric\",\"params\":{\"handleNoResults\":true,\"fontSize\":\"40\"},\"aggs\":[{\"id\":\"1\",\"enabled\":true,\"type\":\"avg\",\"schema\":\"metric\",\"params\":{\"field\":\"system.memory.total\",\"customLabel\":\"Total Memory\"}}],\"listeners\":{}}",
```
Comment (ruflin) [inline metricbeat/etc/kibana/visualization/Avg.-Total-Memory.json]: The file name with `Avg.` is interesting, but that is more something related to Kibana and brings up the question if we should use abbreviations in the title.

**Comment by ruflin** [comment]
LGTM. @monicasarbu elastic/kibana#7789 seems to be merged now.

**Comment by monicasarbu** [comment]
Another blocker for this PR: https://github.com/elastic/kibana/issues/7828

File: metricbeat/etc/kibana/visualization/Avg.-Total-Memory.json (hunk 1 above)
Comment (monicasarbu) [inline metricbeat/etc/kibana/visualization/Avg.-Total-Memory.json]: I agree. I removed it from the title.

**Pull request merged**